src/components/NoteContents/NoteContents.js

```diff
@@ -76,7 +76,7 @@ class NoteContents extends React.Component {
     const { isEditing, closeEditing, renderContents, t, contents } = this.props;
 
     return (
-      <div className="NoteContents" onClick={e => e.stopPropagation()} onMouseDown={e => e.stopPropagation()}>
+      <div className="NoteContents" onClick={isEditing ? e => e.stopPropagation():()=>{}} onMouseDown={isEditing ? e => e.stopPropagation():()=>{}}>
```

Comment:
Yeah you are right it's getting overwritten by another commit. I noticed that in the same file we have a function `handleNoteContentsClick` which does the same thing. Do you mind using that function instead? Reason is that there's some comment which explains the intention of doing so. Thanks!